sum.js

```diff
@@ -0,0 +1,13 @@
+function sumOfNumbers(numbers) {
```

Comment:
please add documentation

Reply:
resolved

Reply:
not satisfied also add comments where required